packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.js

```diff
@@ -176,26 +222,27 @@ const ErrorFormTooltip = ({
         onShown(false);
       }
     };
-
     window.addEventListener("keydown", handleTab);
     return () => {
       window.removeEventListener("keydown", handleTab);
     };
-  }, [onShown, isHelp]);
+  }, [onShown, isHelp, shown, tooltipRef, update]);
 
   return (
+    /* $FlowFixMe: https://github.com/popperjs/react-popper/issues/318 */
     <StyledFormFeedbackTooltip
-      ref={tooltipRef}
+      id={id}
+      ref={setTooltipRef}
       inputSize={inputSize}
-      position={preferredPosition}
-      shown={shown && dimensions.set}
+      shown={shown}
       isHelp={isHelp}
       data-test={dataTest}
-      inlineLabel={inlineLabel}
       aria-live="polite"
-      id={id}
-      {...dimensions}
+      placement={attrs.popper && attrs.popper["data-popper-placement"]}
+      {...popper}
```

Comment:
Be explicit instead so that you can remove `$FlowFixMe` because you already know what you need.
```suggestion
      top={popper.top}
      left={popper.left}
      position={popper.position}
      bottom={popper.bottom}
      right={popper.right}
      transform={popper.transform}
```